lib/wallaroo/core/sink/connector_sink/connector_sink.pony

```diff
@@ -184,13 +208,19 @@ actor ConnectorSink is Sink
     initializer.report_created(this)
 
   be application_created(initializer: LocalTopologyInitializer) =>
-    _mute_upstreams()
     initializer.report_initialized(this)
 
   be application_initialized(initializer: LocalTopologyInitializer) =>
     _initial_connect()
 
   be application_ready_to_work(initializer: LocalTopologyInitializer) =>
+    if _notify.twopc_txn_id_last_committed is None then
```

Comment:
Could this if block be a method on `_notify`?

Reply:
Sure